src/base/bittorrent/session.cpp

```diff
@@ -4918,14 +4929,13 @@ void Session::createTorrent(const lt::torrent_handle &nativeHandle)
                 exportTorrentFile(torrentInfo, torrentExportDirectory(), torrent->name());
             }
         }
-
-        if (isAddTrackersEnabled() && !torrent->isPrivate())
-            torrent->addTrackers(m_additionalTrackerList);
```

Comment:
This broke "Automatically add these trackers..." for torrents added without metadata.

Reply:
> This broke "Automatically add these trackers..." for torrents added without metadata.

Thank you! Will fix it ASAP.

Reply:
> > This broke "Automatically add these trackers..." for torrents added without metadata.
> 
> Thank you! Will fix it ASAP.

Now it should be fixed by #17017.